### PR TITLE
Feat: Multisig details

### PR DESCRIPTION
**Pull request by tuul-wq**
### Fix:
- Show appropriate block explorers in Multisig Details
- Removed unused functions

### New:
- Finally made a standalone Popover with explorers that can wrap any DOM element
- Created 2 functions to produce explorer link for account / extrinsic (with tests)
- Added Wallet types similar to Account types
- WalletConnect wallet have runtime property `isConnected` that is calculated only once in model
- 2 new components `WalletCardLg` `WalletCardMd`, they do not include explorer popover, if you need so use component composition and provide required data (explorers, matrix, custom actions)
- `ExplorerLink` component
image

### CODE REVIEW

**Comment by tuul-wq** [comment]
Task linked: [CU-862kcvq47 Show Multisig wallet details](https://app.clickup.com/t/862kcvq47)

**Comment by tuul-wq** [comment]
Task linked: [CU-8692y90xq WalletCard](https://app.clickup.com/t/8692y90xq)

**Comment by tuul-wq** [comment]
Task linked: [CU-86930u98c Multisig details. Multisig address is displayed in your wallets instead of the signer's address.](https://app.clickup.com/t/86930u98c)

File: src/renderer/features/wallets/WalletSelect/ui/WalletSelect.tsx [hunk 1]
```diff
@@ -44,7 +44,13 @@ export const WalletSelect = ({ action }: Props) => {
             leaveFrom="opacity-100 translate-y-0"
             leaveTo="opacity-0 translate-y-1"
           >
-            <WalletPanel action={action} onClose={close} />
+            <WalletPanel
+              action={action}
+              onClose={() => {
+                close();
+                walletSelectModel.events.clearData();
+              }}
```
Comment (sokolova-an) [inline src/renderer/features/wallets/WalletSelect/ui/WalletSelect.tsx:52]: Maybe move it to a separate function?

File: src/renderer/entities/wallet/ui/ExplorersPopover/ExplorersPopover.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
+import { Popover, Transition } from '@headlessui/react';
+import { PropsWithChildren, ReactNode, MouseEvent, useRef } from 'react';
+
+import { cnTw, getAccountExplorer, toAddress, copyToClipboard } from '@renderer/shared/lib/utils';
+import type { Address, Explorer, AccountId } from '@renderer/shared/core';
+import { ExplorerLink, FootnoteText, IconButton } from '@renderer/shared/ui';
+import { useI18n } from '../../../../app/providers';
```
Comment (sokolova-an) [inline src/renderer/entities/wallet/ui/ExplorersPopover/ExplorersPopover.tsx:7]: import from `@renderer/app/providers` looks better

File: src/renderer/shared/lib/utils/chains.ts [hunk 3]
```diff
@@ -7,3 +9,30 @@ export const toLocalChainId = (chainId?: ChainId): string | undefined => {
...
+
+/**
+ * Get block explorer URL by AccountId or Address
+ * @param explorer explorer with links
+ * @param params address or accountId with addressPrefix
+ * @return {String | undefined}
+ */
+type WithAddress = { address: Address };
+type WithAccountId = { value: Address | AccountId; addressPrefix?: number };
+export const getAccountExplorer = (explorer: Explorer, params: WithAddress | WithAccountId): string | undefined => {
```
Comment (Asmadek) [inline src/renderer/shared/lib/utils/chains.ts:21]: Will this order works fine? Maybe better to mode types little bit higher?

**Review by Asmadek — APPROVED** [review]

**Review by sokolova-an — APPROVED** [review]

**Review by egor0798 — APPROVED** [review]

**Pull request merged**